charts/openshift-metering/templates/presto/presto-coordinator-config.yaml

```diff
@@ -30,7 +30,7 @@ data:
     internal-communication.https.truststore.path=/opt/presto/tls/truststore.pem
 {{- else }}
     http-server.http.port=8080
-    discovery.uri=http://presto:8080
+    discovery.uri=http://presto-coordinator-0:8080
```

Comment:
Wait I thought from the Slack discussion that moving this host name, while it seems like it fixes the Presto announce problem, won't actually work for the cluster members reaching Presto?

Reply:
I threw this PR solely for reference and getting a baseline for Metering installs. Going forward, we need to add some templating around the hive/presto s3 configurations (and avoid hardcoding any bucket regions) and figure out the DNS issues around the Presto discovery URI.

Reply:
👍